src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagerGetTaskRolePermissionsConsumerTest.java

```diff
@@ -0,0 +1,91 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.consumer.wa;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.dsl.DslPart;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import com.google.common.collect.ImmutableMap;
+import org.apache.http.client.fluent.Request;
+import org.junit.jupiter.api.Test;
+import org.springframework.context.annotation.Import;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ContextConfiguration;
+import uk.gov.hmcts.reform.wataskmanagementapi.SpringBootContractBaseTest;
+import uk.gov.hmcts.reform.wataskmanagementapi.auth.permission.entities.PermissionTypes;
+import uk.gov.hmcts.reform.wataskmanagementapi.provider.service.CamundaConsumerApplication;
+import uk.gov.hmcts.reform.wataskmanagementapi.provider.service.TaskManagementProviderTestConfiguration;
+
+import java.io.IOException;
+import java.util.Map;
+
+import static io.pactfoundation.consumer.dsl.LambdaDsl.newJsonBody;
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@PactTestFor(providerName = "wa_task_management_api_task_role_permissions_by_task_id", port = "8991")
+@ContextConfiguration(classes = {CamundaConsumerApplication.class})
+@Import(TaskManagementProviderTestConfiguration.class)
+public class TaskManagerGetTaskRolePermissionsConsumerTest extends SpringBootContractBaseTest {
+
+    public static final String CONTENT_TYPE = "Content-Type";
```

Comment:
minor thing but i  guess we could re-use everywhere if placed in SpringBootContractBaseTest